packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -51,12 +55,20 @@ const unpublishGranule = async ({
   pgCollection,
   granulePgModel = new GranulePgModel(),
   granuleDynamoModel = new models.Granule(),
+  removeGranuleFromCmrFunction = _removeGranuleFromCmr(),
```

Comment:
Do we want to be calling this function here? If so, we need to provide 2 arguments.

Reply:
Good catch: 19d5b71b5e